packages/browser-integration-tests/suites/public-api/debug/test.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+import type { ConsoleMessage } from '@playwright/test';
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../utils/fixtures';
+
+sentryTest('logs debug messages correctly', async ({ getLocalTestUrl, page }) => {
+  const bundleKey = process.env.PW_BUNDLE || '';
+  const hasDebug = !bundleKey.includes('_min');
+
+  const url = await getLocalTestUrl({ testDir: __dirname });
+
+  const consoleMessages: string[] = [];
+
+  page.on('console', (msg: ConsoleMessage) => {
+    consoleMessages.push(msg.text());
+  });
+
+  await page.goto(url);
+
+  await page.evaluate(() => console.log('test log'));
+
+  expect(consoleMessages).toEqual(
+    hasDebug
+      ? [
+          'Sentry Logger [log]: Integration installed: InboundFilters',
+          'Sentry Logger [log]: Integration installed: FunctionToString',
+          'Sentry Logger [log]: Integration installed: TryCatch',
+          'Sentry Logger [log]: Integration installed: Breadcrumbs',
+          'Sentry Logger [log]: Global Handler attached: onerror',
+          'Sentry Logger [log]: Global Handler attached: onunhandledrejection',
+          'Sentry Logger [log]: Integration installed: GlobalHandlers',
+          'Sentry Logger [log]: Integration installed: LinkedErrors',
+          'Sentry Logger [log]: Integration installed: Dedupe',
+          'Sentry Logger [log]: Integration installed: HttpContext',
+          'Sentry Logger [warn]: Discarded session because of missing or non-string release',
+          'test log',
+        ]
+      : ['[Sentry] Cannot initialize SDK with `debug` option using a non-debug bundle.', 'test log'],
```

Comment:
l: This test is gonna fail whenever we add/remove a default integration or change the ordering. I guess one can argue in both directions whether that's a good thing or not, given that it even sort of tests that the integration is actually enabled by default. The downside is that this isn't strictly the purpose of this test. To get around this, we could use `expect.stringContaining` and only expect specific integrations. But in the end this is logaf-l for me so feel free to ignore :)

Reply:
Yeah, I thought about the same thing, and was also not 100% sure... IMHO it's OK to leave it like this for now, if we get to the point where we add/change default integrations and this becomes annoying we can still replace it with something more generic. IMHO for now it is not bad to be "notified" if the defaults change somehow (and that rarely happens anyhow...!)